cookbooks/aws-parallelcluster-common/resources/efa/efa_redhat8.rb

```diff
@@ -24,6 +24,18 @@
   # do nothing
 end
 
+action :check_efa_support do
```

Comment:
Nitpick. You use the action as a plain ruby function. In fact, you can't return a value from an actual action.
It would be cleaner if you define a function (i.e. private behavior) or if you keep using the action (i.e. public behavior) but set an attribute as the outcome. If you choose the former, you'll need to pass the node as argument.

Reply:
Pushed a new version using the latter